### DISCUSSION TITLE
I need a real person to get me verified

### DISCUSSION THREAD

**Discussion by xuejingji** [community/community#50824] [GitHub Education]
Hi,

I have tried so many times and uploaded my student ID, transcript, and even my degreeWorks portal!
However, the system still won't let me verify I'm a student! I have no idea what I can do now, I have followed all the instructions.
I have created a ticket, but never get manually help. Can I get someone real to get me verified?

Thanks.

**Comment by rwong917** [comment]
I'm having just the same problem. Provided an email, screenshot of Student portal, and a transcript, yet it is still requiring me to upload a student ID which I do not have. I submitted a ticket 12 days ago and still haven't heard back.

**Comment by saaxena** [comment]
You should use your college email and then upload the student id it will work  , same has worked for me and i was verified in less than 10 minutes . Hope it helps

**Reply by lyk7539511** [reply]
> You should use your college email and then upload the student id it will work , same has worked for me and i was verified in less than 10 minutes . Hope it helps

I followed your answer, but also receive a rejection letter,
This is my rejection reason from GitHub Education <edu-noreply@github.com>:
---
Unfortunately, we weren't able to approve your educational discount request for [@lyk7539511](https://github.com/lyk7539511).
What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
---
My school is real, and I am not a student who learned on the web.
I can't think of any way to solve this problem, do you have any ideas?

**Reply by jge162** [reply]
So you live in `Georgia?` They do not see that is why you are being rejected based on the response. @xuejingji

**Reply by saaxena** [reply]
Use Same Name on GitHub as per your college id  and connect college email to GitHub and then try .  I am currently in India and my verification completed in less than 10 minutes

**Reply by jge162** [reply]
But your school was also in India correct? @saaxena I think @xuejingji issue might be she maybe does not live in Georgia. they are able to see on Github.

**Reply by saaxena** [reply]
Yes

**Reply by jge162** [reply]
so for you it is easy. @xuejingji needs to be on the same IP as her school or near it.  She appears not to be based on GitHubs checks.

**Reply by xuejingji** [reply]
> So you live in `Georgia?` They do not see that is why you are being rejected based on the response. @xuejingji

Yes, I live in GA

**Reply by xuejingji** [reply]
> Use Same Name on GitHub as per your college id and connect college email to GitHub and then try . I am currently in India and my verification completed in less than 10 minutes

I have done this and tried several times, not working 😭

**Reply by xuejingji** [reply]
> But your school was also in India correct? @saaxena I think @xuejingji issue might be she maybe does not live in Georgia. they are able to see on Github.

I live in GA and my college is in GA too…

**Comment by RuiStarlit** [comment]
same as you

**Comment by ranpin** [comment]
me too

**Comment by samus-aran** [comment]
Hi @xuejingji 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by xuejingji** [reply]
I just tried again and it works now. Thank you so much!
